spec/solitaire_spec.rb

```diff
@@ -1,23 +1,21 @@
 require 'spec_helper'
-require_relative '../solitaire-v1'
+require_relative '../solitaire'
 
-describe Solitaire, '#encrypt_message' do
-  it 'successfully encrypts a message' do
-    message = 'Code in Ruby, live longer!'
+describe Solitaire,"#encrypt" do
+  it 'successfully encrypts message with unkeyed deck' do
     expected_output = 'GLNCQ MJAFF FVOMB JIYCB'
 
-    encrypted_message = Solitaire.new(message).encrypt_message
+    encrypted_message = Solitaire.new(message: 'Code in Ruby, live longer!').encrypt
```

Comment:
I would keep the `message` temporary variable string, instead of deleting it and being explicit here. It cuts down on the git diff and makes it easier to see what's been changed.